kotsadm/web/src/components/apps/DashboardCard.jsx

```diff
@@ -375,7 +387,20 @@ export default class DashboardCard extends React.Component {
                         <div className="flex flex-column">
                           {startSnapshotErr &&
                             <p className="u-color--chestnut u-fontSize--small u-fontWeight--medium u-lineHeight--normal flex">{startSnapshotErrorMsg}</p>}
-                          <span className="card-link flex" onClick={startManualSnapshot}> Start a snapshot </span>
+                          <div className="flex u-position--relative">
+                            <Select
+                              className="replicated-select-container snapshot"
+                              classNamePrefix="replicated-select"
+                              options={startSnapshotOptions}
+                              getOptionLabel={(option) => option.name}
+                              getOptionValue={(option) => option.name}
+                              value={selectedSnapshotOption}
+                              onChange={onSnapshotOptionChange}
+                              isOptionSelected={(option) => { option.name === selectedSnapshotOption.name }}
+                              styles={customStyles}
```

Comment:
Do we need to add the prop on this to make sure you can't type in the select? I think it's `isSearchable` or something like that.

Reply:
great point!